tests/lib/rules/no-param-reassign.js

```diff
@@ -91,6 +91,13 @@ ruleTester.run("no-param-reassign", rule, {
             code: "function foo(a) { for (bar of baz) a.b; }",
             options: [{ props: true }],
             parserOptions: { ecmaVersion: 6 }
+        },
+        {
+            code: "function foo(bar, baz) { bar.a = true; baz.b = false; }",
+            options: [{
+                ignorePropertyModificationsForRegex: ["^(foo|bar)$"],
+                ignorePropertyModificationsFor: ["baz"]
+            }]
         }
```

Comment:
I think that this test should have `props: true`, otherwise the code would be valid regardless of the ignore arrays?

It looks like the `schema` in this rule is configured to catch some incorrect settings (explicit `props: false`+ arrays), but doesn't catch implicit `props: false` + arrays.

Reply:
This should be fixed. I just added the option, please let me know if there's anything else I need to do here.